include/ccf/pal/attestation_sev_snp.h

```diff
@@ -75,13 +75,28 @@ QPHfbkH0CyPfhl1jWhJFZasCAwEAAQ==
       ecdsa_p384_sha384 = 1
     };
 
+#pragma pack(push, 1)
+    // Table 8
+    struct GuestPolicy
+    {
+      uint8_t abi_minor;
+      uint8_t abi_major;
+      uint8_t smt : 1;
+      uint8_t reserved : 1;
+      uint8_t migrate_ma : 1;
+      uint8_t debug : 1;
+      uint8_t single_socket : 1;
+      uint64_t reserved2 : 43;
+    };
+#pragma pack(pop)
+
 #pragma pack(push, 1)
     // Table 21
     struct Attestation
     {
       uint32_t version; /* 0x000 */
       uint32_t guest_svn; /* 0x004 */
-      uint64_t policy; /* 0x008 */
+      struct GuestPolicy policy; /* 0x008 */
```

Comment:
```suggestion
      struct GuestPolicy guest_policy; /* 0x008 */
```

Reply:
I'm happy to do this, the only risk is then we diverge from the AMD docs?

Reply:
Absolutely, let's not deviate from the official docs. I was just trying to avoid confusion with the `host_data` field that's the hash of a security policy.